2020-07-30 15:40:16.097 | ERROR    | checker.check:test_proxy:40 - Cannot connect to host 192.168.1.3:7777 ssl:None [Connect call failed ('192.168.1.3', 7777)]
2020-07-30 15:40:18.109 | ERROR    | checker.check:test_proxy:40 - Cannot connect to host 192.168.1.1:8888 ssl:None [Connect call failed ('192.168.1.1', 8888)]
2020-07-30 15:40:35.096 | ERROR    | checker.check:test_proxy:40 - Cannot connect to host 192.168.1.2:6666 ssl:None [Connect call failed ('192.168.1.2', 6666)]
2020-07-30 15:49:25.878 | ERROR    | checker.check:test_proxy:40 - Cannot connect to host 192.168.1.3:7777 ssl:None [Connect call failed ('192.168.1.3', 7777)]
2020-07-30 15:49:27.890 | ERROR    | checker.check:test_proxy:40 - Cannot connect to host 192.168.1.1:8888 ssl:None [Connect call failed ('192.168.1.1', 8888)]
2020-07-30 15:49:44.876 | ERROR    | checker.check:test_proxy:40 - Cannot connect to host 192.168.1.2:6666 ssl:None [Connect call failed ('192.168.1.2', 6666)]
2020-07-30 16:02:14.348 | ERROR    | checker.check:test_proxy:40 - Cannot connect to host 192.168.1.3:7777 ssl:None [Connect call failed ('192.168.1.3', 7777)]
2020-07-30 16:02:16.355 | ERROR    | checker.check:test_proxy:40 - Cannot connect to host 192.168.1.1:8888 ssl:None [Connect call failed ('192.168.1.1', 8888)]
2020-07-30 16:02:33.348 | ERROR    | checker.check:test_proxy:40 - Cannot connect to host 192.168.1.2:6666 ssl:None [Connect call failed ('192.168.1.2', 6666)]
